GettingStartedGuide.md

```diff
@@ -71,14 +73,18 @@ After you have followed the instructions in the AWS IoT Core Setup Guide, you wi
 ### 2.2 Configure the project with the AWS IoT Thing Name and AWS device Endpoint
 
 The demo will connect to the AWS IoT device Endpoint that you configure here.
+> Note: remember to set your build target using `idf.py set-target <target>`
 
 1. From a terminal/command prompt, run `idf.py menuconfig`. This assumes the ESP-IDF environment is exported-- i.e. that export.bat/export.sh, which can be found under the ESP-IDF directory, has been run, or that you are using the ESP-IDF command prompt/terminal. For Visual Studio (VS) Code users who are using the Espressif IDF extension, do ->View->Command Palette->Search for `ESP-IDF: SDK Configuration editor (menuconfig)` and select the command. The `SDK Configuration editor` window should pop up after a moment.
    (Note: If running menuconfig from within a VS Code command prompt, 'j' and 'k' may have to be used in place of the 'up' and 'down' arrow keys. Alternately, one can use a command prompt/terminal outside of the VS Code editor).
 2. Select `Featured FreeRTOS IoT Integration` from the menu.
 3. Set `Endpoint for MQTT Broker to use` to your **AWS device Endpoint**.
 4. Set `Port for MQTT Broker to use` to `8883`.
 5. Set `Thing name` to your **Thing Name**.
-6. Go back to main menu, Save and Exit.
+6. If you are using the `ESP32` chip you may experience buffer overflow, feel free to decrease the task's buffer size of the demo or `coreMQTT-Agent network buffer size` to fit your use case.
+7. You may also increase the `Timeout for receiving CONNACK in milliseconds` if you are experiencing timeout issues.
+8. Furthermore you may want to use the Amazon's FreeRTOS that can be enabled from `Component config > FreeRTOS > Kernel> Run the Amazon SMP FreeRTOS kernel instead (FEATURE UNDER DEVELOPMENT)`
```

Comment:
Think this line should be "If you want to use the FreeRTOS-Kernel's Symmetrical multiprocessing feature..."?

Reply:
noted, thank you